src/components/Nav/TopNav.jsx

```diff
@@ -78,7 +78,7 @@ const TopNav = () => {
       <Toolbar className={clsx(isMobile && classes.mobileTopMenu)}>
         <Typography
           component="a"
-          href={PAGE_ROUTES[MISSING]}
+          href={"/"}
```

Comment:
instead of changing `PAGE_ROUTES[MISSING]` to `/`, you could have changed it in our `constants.js` file to be more efficient